### DISCUSSION TITLE
Security and Production best practices section in Docs

### DISCUSSION THREAD

**Discussion by ivanjeremic** [fastify/fastify#4310] [Ideas] [answered]
Express has this nice section and I was not able to find something similar in Fastify docs (Correct me if I'm wrong), but is would be super helpful to have a section which guides you trough running a successful production server on your own which is secure. [https://expressjs.com/en/advanced/best-practice-security.html](https://expressjs.com/en/advanced/best-practice-security.html)

**Comment by jsumners** [comment, marked as answer]
I really don't think it's our responsibility to teach common development practices. We have enough burden maintaining the libraries and documentation around them.

That being said, I do understand how you could say "but the recommendations doc". In that case, we routinely get questions about how to run Fastify applications with things like `pm2`. So we had to write up our recommendations to answer this continually recurring question.
